### ISSUE TITLE
Toggle button group not conveying state

### ISSUE THREAD

**Issue by sirrah-tam** [ithaka/pharos#430]
**Expected behavior**
The [Toggle Button Group](https://pharos.jstor.org/components/toggle-button-group) conveys the active/inactive status of each control both visually and programmatically.

**Actual behavior**
These controls do not programmatically convey their current state and are only announced with name and button role, ex. "List, button; Grid, button". Screen reader users will not know which option is currently selected.

**Steps to reproduce the issue**

1. While using a screen reader, navigate to the List/Grid control
2. Navigate between controls
3. Observe announcement of Name and Role, but does not announce state.

**Screenshots or code**
<img width="1333" alt="Screen Shot 2022-10-20 at 11 26 12 AM" src="https://user-images.githubusercontent.com/6874453/197029039-98e2a170-a5b7-4607-9c0b-c3e572944f76.png">

**Pharos version**
Production

**Your environment**
OS: macOS, Windows
Browser: Chrome, Firefox, Safari
Version: 97.0, 105.0.2, 16.0

**Additional information**
Developers must ensure that the active/inactive or selected state of a control is conveyed programmatically. Most semantic input elements have this feature built in, checkbox, radio button, etc. Recommendation would be to treat this control as radio buttons, keep the same visual styling, but change the inputs accordingly. If this is not possible, or for future custom controls, developers can use the aria-selected attribute to convey this information specifically for users of assistive technology.

**Comment by sirrah-tam** [comment]
@daneah - One side affect of the change we made to this is that all Pharos buttons now have the `aria-pressed="undefined"` attribute added by default. While this isn't necessarily problematic at first, it has shown a bit of inconsistency across some browser & AT combinations. While testing with VoiceOver on Safari, the presence of that attribute makes the button be parsed as a "toggle button" and is announced as such even though I don't believe that's the intention. Users will expect certain behavior if its announced as that type of button control, and could become confused if it does something different.
